dev-common-postgres-image/init.sql

```diff
@@ -596,3 +596,8 @@ COMMIT;
 -- Export File Templates
 
 -- Email Template
+-- HMS Email Template
+INSERT INTO casev3.email_template (pack_code, description, notify_template_id, notify_service_ref, metadata, template) VALUES
+('MNE_EN_HMS', 'Main Notification Email', '883cee97-7a41-4fdb-8a09-0972c86b9375', 'Office_for_National_Statistics_surveys_NHS', null, '["__uac__", "PORTAL_ID", "COLLEX_OPEN_DATE", "COLLEX_CLOSE_DATE", "FIRST_NAME", "__sensitive__.LAST_NAME"]'),
+('MRE_EN_HMS', 'Main Reminder Email', '1b46b5e1-e247-48b3-b3b3-bf949efd79cb', 'Office_for_National_Statistics_surveys_NHS', null, '["__uac__", "PORTAL_ID", "COLLEX_OPEN_DATE", "COLLEX_CLOSE_DATE", "FIRST_NAME", "__sensitive__.LAST_NAME"]')
```

Comment:
The notify service ref `Office_for_National_Statistics_surveys_NHS` doesn't exist. It's something we add as part of the notify config. Currently the config is using `Office_for_National_Statistics_surveys_UKHSA` for the gov notify service we use. It's something we need to change as part of this. If it's NHS branding on the template then we should look to rename it from `Office_for_National_Statistics_surveys_UKHSA` to `Office_for_National_Statistics_surveys_NHS`